doc/whats_new/v0.22.rst

```diff
@@ -127,7 +127,11 @@ Changelog
 
 - |Efficiency| The 'liblinear' logistic regression solver is now faster and
   requires less memory.
-  :pr:`14108`, pr:`14170` by :user:`Alex Henrie <alexhenrie>`.
+  :pr:`14108`, :pr:`14170` by :user:`Alex Henrie <alexhenrie>`.
+
+- |Fix| :class:`linear_model.Ridge` now accepts Fortran arrays and make a
+  conversion instead of failing.
```

Comment:
Precise that it's with solver='sag'. The others were already ok, right ? 

It accepts any memory layout now, not just C or F.